### ISSUE TITLE
Efficiency Ratios

### ISSUE THREAD

**Issue by cbarbaux** [EnergyWater/EMSA#18]
On the efficiency tab of the results, when we can check the main energy efficiency ratio of equipment and process, the unit are missing.
![efficiencyratios](https://user-images.githubusercontent.com/34023517/33558661-c039bb14-d90a-11e7-8cd1-45254fc99db8.png)
Also, the difference between the four ratios are not clear, some explanations have to be added

**Comment by WINGSTeam** [comment]
> On the efficiency tab of the results, when we can check the main energy efficiency ratio of equipment and process, the unit are missing.

We have updated some of the charts with the units that we found at the indicators excel provided. 

> Also, the difference between the four ratios are not clear, some explanations have to be added

Maybe ITCL or Socamex can help with this.

**Comment by AaronITCL** [comment]
Please @cbarbaux , follow the instructions set for the Github at the time of creating an issue, we are missing the Milestone and the Labels.

> Also, the difference between the four ratios are not clear, some explanations have to be added

We will consider this as a proposal, so for now will not be prioritary.

**Comment by cbarbaux** [comment]
Thanks @WINGSTeam for the modification.

@AaronITCL I've tried to put some labels on the issue. However I strongly disagree with your last comment. The ratios are not self-explanatory and because there are no further information in the user manual, we need some guidance in the tool. 
Users need to know what you mean by "Efficiency ratio optimum", "Energy ratio nominal", "Energy Efficiency ratio" and "Oversize ratio".

**Comment by AaronITCL** [comment]
New issue appears in the EMSA, we miss some process results, as you can see in the pictures:
_ScreenShot of the View button of the scenarios module:_

![image](https://user-images.githubusercontent.com/33861863/33830726-96acb8b2-de75-11e7-8ea7-dc2d3f9e641c.png)

_ScreenShot of the results module._

![image](https://user-images.githubusercontent.com/33861863/33830768-cd90e4b6-de75-11e7-925f-9e9664283f0c.png)

On the other hand attending to this:

> Users need to know what you mean by "Efficiency ratio optimum", "Energy ratio nominal", "Energy Efficiency ratio" and "Oversize ratio".

Please @cbarbaux, could you send us a proposal?

**Comment by david-ITCL** [comment]
@WINGSTeam , could you please prioritize this task to be ready for tomorrow's webinar?

**Comment by cbarbaux** [comment]
@WINGSTeam  here are my notes from a conversation with ITCL regarding the meaning of the different ratios:

- Energy Efficiency ratio: Ratio as calculated for the company

- Efficiency ratio nominal: Best theoretical ratio

- Efficiency ratio optimum: Best ratio from benchmark

- Oversize ratio: comparison of the installed power (real and nominal)
@david-ITCL can you confirm?

**Comment by WINGSTeam** [comment]
> New issue appears in the EMSA, we miss some process results, as you can see in the pictures:
> ScreenShot of the View button of the scenarios module:

The issue of the Heating process should be fixed now

**Comment by david-ITCL** [comment]
@cbarbaux , let me give some clarification about the ratios:

- Energy Efficiency ratio: Ratio as calculated for the company (calculated with real working conditions from the equipment --> measured, estimaded, calculated)

- Efficiency ratio nominal: **Ratio according design conditions** (calculated with manufacturer's data from the equipment, not real working data)
 
- Efficiency ratio optimum: Best ratio (best possible ratio for the working conditions, sometimes it is an internal equation but, if we cannot calculate it, this will come from the benmarking)

- Oversize ratio: comparison of the installed power **(nominal vs optimum)**

**Comment by WINGSTeam** [comment]
@david-ITCL , @cbarbaux : Should we include some lines explaining these ratios at the front end ?

**Comment by david-ITCL** [comment]
@WINGSTeam @cbarbaux @EnergyWater/admin @EnergyWater/viewer 
Regarding the feedback, it's clear that we need more guidance in the tool, but, as we have mentioned before, this will be an improvement that we will adress later, once EMSA v.1 is complete. Now, we must prioritize the functionality that is included in the user manual and we still miss.
After that, hopefully next week, one of the first imporvements could be adding those explanations both on the tool and the user manual, but we have to be efficient on that tasks, first gather all the information that partners think is needed and then find a way to do it.
In that aspect, we think both in two methods to include
- First, add an "info" button and display key information such as those indicators that you mention
- Second, add short videos (1 min) as know pills that explain certain processes like creating an scenario or entering information.

Now, we can start gathering all guidance needed, to do this, we need the partners to test the tool and tell us what kind of guidande do they need, later, once this is defined, we will implement it.
To do this, we can open one issue per module related to guidance, now, we have only received guidande needs for Scenarios module, so that's why there is only an open issue regarding it.

**Comment by WINGSTeam** [comment]
Thanks @david-ITCL for the explanation, then we could probably close this issue and reopen it when appropriate, in order to be easier for the partners to follow the issues and discussions.
